sway/config.c

```diff
@@ -358,6 +358,7 @@ static void config_defaults(struct sway_config *config) {
 	config->blur_xray = false;
 	config->blur_params.num_passes = 2;
 	config->blur_params.radius = 5;
+	config->blur_params.noise = 0.02;
```

Comment:
Why is it such a low value?

Reply:
Only a little noise is needed to clear up banding, a large amount of noise can be undesirable visually. `0.02` is a value I personally like, but hyprland's default is even lower than this